config/initializers/flipper.rb

```diff
@@ -12,13 +12,10 @@
 
 Flipper.configure do |config|
   config.default do
-    activerecord_adapter = Flipper::Adapters::ActiveRecord.new
-    cache = ActiveSupport::Cache::MemoryStore.new
-    # Flipper settings will be stored in postgres and cached in memory for 1 minute in production/staging
-    cached_adapter = Flipper::Adapters::ActiveSupportCacheStore.new(activerecord_adapter, cache, expires_in: 1.minute)
-    adapter = Rails.env.development? || Rails.env.test? ? activerecord_adapter : cached_adapter
+    adapter = Flipper::Adapters::ActiveRecord.new
+    cache = Rails.cache
+    adapter = Flipper::Adapters::ActiveSupportCacheStore.new(adapter, cache, expires_in: 5.minutes)
```

Comment:
For comparison purposes, while gathering metrics, should we keep this `expires_in` to 1 minute to match the old caching method? (Might make sense to move it up to 5, or higher, eventually but just for an equal comparison)